Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -393,6 +430,14 @@ void MemWatchWidget::pasteWatchFromClipBoard(const QModelIndex& referenceIndex)
   m_hasUnsavedChanges = true;
 }
 
+void MemWatchWidget::copyAddressToClipboard(u32 addr)
+{
+  char hex_string[8];
+  snprintf(hex_string, 8, "%X", addr);
+  QClipboard* clipboard = QApplication::clipboard();
+  clipboard->setText(hex_string);
```

Comment:
The use of the raw C array and `snprintf()` is error prone. Currently the array of size 8 could not possibly hold 8 characters _plus_ the null character (`\0`); garbage may be put in the clipboard.

Suggestion:
```cpp
const QString text{QString::number(addr, 16).toUpper()};
QApplication::clipboard()->setText(text);
```

Reply:
That makes sense. This is also simpler. With this I don't think I need a separate function to perform this, so I am going to integrate it into the context menu itself rather than be a separate method.